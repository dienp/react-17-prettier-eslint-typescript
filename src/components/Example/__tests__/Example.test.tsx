import { render, screen } from '@testing-library/react';
import { Example } from '../Example';

describe('<Example />', () => {
  it('should match snapshot', () => {
    const { container } = render(<Example />);
    expect(container.firstChild).toMatchSnapshot();
  });
  it('should render', () => {
    render(<Example />);
    const linkElement = screen.getByText(/Example/i);
    expect(linkElement).toBeInTheDocument();
  });
});
