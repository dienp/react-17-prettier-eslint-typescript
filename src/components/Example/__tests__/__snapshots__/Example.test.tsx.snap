// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> should match snapshot 1`] = `
<div
  data-test-id="Example"
>
  Example
</div>
`;
